Cargo.toml

```diff
@@ -37,3 +37,5 @@ default = []
 # This allows data to be read into a Value and written back to a JSON string
 # while preserving the order of map keys in the input.
 preserve_order = ["linked-hash-map"]
+
+arbitrary_precision = []
```

Comment:
Please add a comment explaining what is (and maybe what is not) guaranteed when you enable this feature.

Reply:
Something like the following?